Index: new_seed.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/new_seed.php b/new_seed.php
--- a/new_seed.php	(revision 3f673d2042f4a1fdd15aafb56b3347e871090bce)
+++ b/new_seed.php	(date 1690564034443)
@@ -1,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Насіння</title>
+</head>
+<body>
+<?php
+if (isset($_COOKIE['id']) && isset($_COOKIE['hash'])) {
+    $link = mysqli_connect("localhost", "root", "admin", "Garden");
+
+    // Get all the cultures from the database
+    $culturesResult = mysqli_query($link, "SELECT * FROM Culture");
+    $cultures = mysqli_fetch_all($culturesResult, MYSQLI_ASSOC);
+
+    if (isset($_POST['back'])) {
+        header("Location: seed_main.php");
+    } elseif (isset($_POST['home_page'])) {
+        header("Location: home_page.php");
+    }
+
+    if (isset($_POST['submit'])) {
+        $idSeed = $_POST['idSeed'];
+        $nameSeed = $_POST['nameSeed'];
+        $idCulture = $_POST['culture'];
+
+        // Insert the seed into the Seed table
+        mysqli_query($link, "INSERT INTO Seed (Id_Seed, Name_Seed, ID_Culture) VALUES ($idSeed, '$nameSeed', $idCulture)");
+    }
+}
+?>
+
+<form method="post">
+    <label>ID Насіння:
+        <input type="number" name="idSeed" required>
+    </label>
+    <br>
+    <label>Назва насіння:
+        <input type="text" name="nameSeed" required>
+    </label>
+    <br>
+    <label>Культура:
+        <select name="culture">
+            <?php foreach($cultures as $culture): ?>
+                <option value="<?php echo $culture['Id_Culture']; ?>">
+                    <?php echo $culture['Name_Culture']; ?>
+                </option>
+            <?php endforeach; ?>
+        </select>
+    </label>
+    <br>
+    <button name="submit" type="submit" >Зберегти</button>
+</form>
+
+<form method="POST">
+    <input name="back" type="submit" value="Назад">
+    <input name="home_page" type="submit" value="На головну сторінку">
+</form>
+</body>
+</html>
Index: culture_main.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/culture_main.php b/culture_main.php
--- a/culture_main.php	(revision 3f673d2042f4a1fdd15aafb56b3347e871090bce)
+++ b/culture_main.php	(date 1690562928003)
@@ -1,0 +1,40 @@
+<?php
+
+$link = mysqli_connect("localhost", "root", "admin", "Garden");
+
+if (isset($_COOKIE['id']) && isset($_COOKIE['hash'])) {
+    if (isset($_POST['add'])) {
+        header("Location: new_culture.php");
+    } elseif (isset($_POST['update'])) {
+        header("Location: update_culture.php");
+    }elseif (isset($_POST['back'])) {
+        header("Location: home_page.php");
+    }
+
+    $result = mysqli_query($link, "SELECT * FROM Culture");
+
+    if (mysqli_num_rows($result) > 0) {
+        echo "<table>";
+        echo "<tr><th>Назва культури</th></tr>";
+
+        while ($row = mysqli_fetch_assoc($result)) {
+            echo "<tr>";
+            echo "<td>" . $row['Id_Culture'] . "</td>";
+            echo "<td>" . $row['Name_Culture'] . "</td>";
+            echo "</tr>";
+        }
+
+        echo "</table>";
+    } else {
+        echo "У вас немає культур.";
+    }
+} else {
+    print "Увімкніть кукі";
+}
+?>
+
+<form method="POST">
+    <input name="add" type="submit" value="Додати нову культуру">
+    <input name="update" type="submit" value="Редагувати культуру">
+    <input name="back" type="submit" value="Назад">
+</form>
\ No newline at end of file
Index: new_culture.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/new_culture.php b/new_culture.php
--- a/new_culture.php	(revision 3f673d2042f4a1fdd15aafb56b3347e871090bce)
+++ b/new_culture.php	(date 1690562702533)
@@ -1,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Культури</title>
+</head>
+<body>
+<?php
+if (isset($_COOKIE['id']) && isset($_COOKIE['hash'])) {
+    $link = mysqli_connect("localhost", "root", "admin", "Garden");
+
+
+
+    if (isset($_POST['back'])) {
+        header("Location: culture_main.php");
+    } elseif (isset($_POST['home_page'])) {
+        header("Location: home_page.php");
+    }
+
+    if (isset($_POST['submit'])) {
+        $idCulture = $_POST['idCulture'];
+        $cultureName = $_POST['cultureName'];
+
+        // Insert the culture into the Culture table
+        mysqli_query($link, "INSERT INTO Culture (Id_Culture, Name_Culture) VALUES ($idCulture, '$cultureName')");
+    }
+}
+?>
+
+<form method="post">
+    <label>ID Культури:
+        <input type="number" name="idCulture" required>
+    </label>
+    <br>
+    <label>Назва культури:
+        <input type="text" name="cultureName" required>
+    </label>
+    <br>
+    <button name="submit" type="submit" >Зберегти</button>
+</form>
+
+<form method="POST">
+    <input name="back" type="submit" value="Назад">
+    <input name="home_page" type="submit" value="На головну сторінку">
+</form>
+</body>
+</html>
Index: seed_main.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/seed_main.php b/seed_main.php
--- a/seed_main.php	(revision 3f673d2042f4a1fdd15aafb56b3347e871090bce)
+++ b/seed_main.php	(date 1690563703946)
@@ -1,0 +1,41 @@
+<?php
+
+$link = mysqli_connect("localhost", "root", "admin", "Garden");
+
+if (isset($_COOKIE['id']) && isset($_COOKIE['hash'])) {
+    if (isset($_POST['add'])) {
+        header("Location: new_seed.php");
+    } elseif (isset($_POST['update'])) {
+        header("Location: update_seed.php");
+    } elseif (isset($_POST['back'])) {
+        header("Location: home_page.php");
+    }
+
+    $result = mysqli_query($link, "SELECT * FROM Seed");
+
+    if (mysqli_num_rows($result) > 0) {
+        echo "<table>";
+        echo "<tr><th>Id_Seed</th><th>Name_Seed</th><th>ID_Culture</th></tr>";
+
+        while ($row = mysqli_fetch_assoc($result)) {
+            echo "<tr>";
+            echo "<td>" . $row['Id_Seed'] . "</td>";
+            echo "<td>" . $row['Name_Seed'] . "</td>";
+            echo "<td>" . $row['ID_Culture'] . "</td>";
+            echo "</tr>";
+        }
+
+        echo "</table>";
+    } else {
+        echo "У вас немає насіння.";
+    }
+} else {
+    print "Увімкніть кукі";
+}
+?>
+
+<form method="POST">
+    <input name="add" type="submit" value="Додати нове насіння">
+    <input name="update" type="submit" value="Редагувати насіння">
+    <input name="back" type="submit" value="Назад">
+</form>
